README.md

```diff
@@ -1,58 +1,91 @@
+- Ideally, all code should be checked in alongside tests that validate its
+  correctness. These should include unit tests, at a minimum. Larger features
+  should include integration tests (possibly in parallel PRs, depending on
+  feature size).
+- Use semantic commit messages.
+
+### PR Format
+
+PRs should include:
+
+- __A meaningful and descriptive title, chosen according to naming conventions.__ (TBD)
+- __Links to any relevant tickets or issues.__
+- __A description of "What's Here."__ Describe what you did and _why_. This
+  description should include the previous behavior of the code and how the new
+  behavior differs.
+- __Steps to test.__ Reviewers should be able to run the code themselves in
+  order to explore and validate its behavior.
+- __Questions/What you'd like feedback on.__ Call out any specific areas of the
+  code or concepts that you would like people to focus on.
+- __References.__ Include links to design docs/PRDs, runs of tests/benchmarks,
+  etc.
 
 ### Review Etiquette and Expectations
 
-- Spend the time to read, understand, and preferably test what's in the PR -- time spent here benefits all of us and our users.
-- Ask questions - don't assume that you know why the author made the choices they did.
-- Try to understand the big picture -- where this code is going, how it's meant to serve users and how it will fit into our codebase.
-- Use the PR forum for contructive discussion
-- Always consider perfomance, security, and adherence to existing mental models and patterns.
-- An approval means you're confident that this PR will both do what it's supposed to do, improve the Mozilla Social codebase, and not introduce technical debt.
-- Expect to complete your review within 1 business day of the PR going to review. If you can't do that, please tell the author early on.
-
-### Reviewer Expectations
+- __First and foremost: use the PR forum for _constructive_ discussion.__ Discuss
+  and critique __the code, not the author.__
+- Try to understand the big picture. Where this code is going? How is it meant to serve users? How will it fit into our codebase?
+- Spend the time to read, understand, and preferably test what's in the PR--time
+  spent here benefits all of us and our users. Don't rubber-stamp code, even if
+  it is a small change. Detailed and thorough code review improves our codebase
+  over time.
+- Consider performance, security, and adherence to existing mental models and patterns.
+- Ask questions. Don't assume that you know why the author made the choices they
+  did. If you see an alternative approach, or might prefer code to be written
+  differently, __ask__ the author why they structured something the way that
+  they did--there might be a reason that you don't see.
+- If you require a change to be made or see an issue that should be addressed,
+  __explain__ the _whys_ and your suggested _hows_.
+
+  > __Unhelpful:__ "This is a bug; use `int` or at least make this `unsigned`." 
```

Comment:
I 😍 these changes. These are great, be helpful and ask, not tell.

Reply:
Thanks!